Include/pystate.h

```diff
@@ -75,18 +75,42 @@ typedef struct {
     wchar_t *dll_path;      /* Windows DLL path */
 #endif
 
-    /* Private fields */
-    int _disable_importlib; /* Needed by freeze_importlib */
+    /* If greater than 0, enable isolated mode: sys.path contains
+       neither the script's directory nor the user's site-packages directory.
+
+       Set to 1 by the -I command line option. If set to -1 (default), inherit
+       Py_IsolatedFlag value. */
+    int isolated;
+
+    /* If equal to zero, disable the import of the module site and the
+       site-dependent manipulations of sys.path that it entails. Also disable
+       these manipulations if site is explicitly imported later (call
+       site.main() if you want them to be triggered).
+
+       Set to 0 by the -S command line option. If set to -1 (default), set to
+       the negative value of Py_NoSiteFlag. */
```

Comment:
This comment isn't right, as the field gets set to the result of `!Py_NoSiteFlag`

Reply:
I wrote "negative value of !Py_NoSiteFlag". Is that wrong?

Reply:
"negative value" is ambiguous here because we're dealing with C integers rather than strict booleans, and arithmetic negation (`-value`) and logical negation (`!value`) do different things. Since this is C code, and the English spelling is ambiguous, just including the exact C expression in the comment is clearer than trying to express the operation as words.